src/hvm.c

```diff
@@ -113,9 +116,9 @@ typedef u32 Numb; // Numb ::= 29-bit (rounded up to u32)
 #define G_RBAG_LEN (TPC * RLEN)
 
 typedef struct Net {
-  APair node_buf[G_NODE_LEN]; // global node buffer
-  APort vars_buf[G_VARS_LEN]; // global vars buffer
-  APair rbag_buf[G_RBAG_LEN]; // global rbag buffer
+  APair* node_buf; // global node buffer, size = G_NODE_LEN
```

Comment:
Since large array will cause '[Total size of array must not exceed 0x7fffffff bytes](https://stackoverflow.com/questions/42173040/error-total-size-of-array-must-not-exceed-0x7fffffff-bytes)' on Windows.

Reply:
But that's for static arrays, here Net and GNet are dinamically allocated so this shouldn't apply. 
Was this giving you a compilation error?